modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -36,6 +36,10 @@ header {
 		&:hover {
 			z-index: 1034;
 		}
+
+		.portlet-content-editable:hover {
+			min-width: 184.37px;
```

Comment:
I don't think we can hard code a value like this since each portlet would be a different size and the content's within it could be different.